src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cssDeclarations = sourceString.split(';');
+  const styleObject = {};
+
+  cssDeclarations.forEach((property) => {
```

Comment:
You need to make sure that property is not empty or contains only spaces before dividing by propertyArr.
```suggestion
  cssDeclarations.forEach((property) => {
     if (property.trim()) { 
```